src/magellan_motion/src/path_planner/planner.cpp

```diff
@@ -60,9 +65,9 @@ Path PathPlanner::getPlan(std::shared_ptr<Successor> goalNode) {
 
     while (parent != nullptr) {
         PoseStamped pt;
-        pt.pose.position.x = static_cast<float>(parent->xPose) / floatPrecision;
-        pt.pose.position.y = static_cast<float>(parent->yPose) / floatPrecision;
-        pt.header.frame_id = "odom";
+        pt.pose.position.x = static_cast<float>(parent->xPose) * floatPrecisionDivide;
```

Comment:
the poses being obtained here are all relative to base_link, as planning always starts from (0,0). You need to transform these points into the `transformFrame` frame first